docs/operations/server.md

```diff
@@ -11,140 +11,192 @@ describe how to configure JanusGraph Server to handle HTTP endpoint
 interactions. For information about how to connect to a JanusGraph
 Server from different languages refer to [Connecting to JanusGraph](../interactions/connecting/index.md).
 
-## Getting Started
+## Starting a JanusGraph Server
+
+JanusGraph Server comes packaged with a script called `bin/janusgraph-server.sh` to get it started:
+```txt
```

Comment:
(nitpick) Shouldn't we remove most of the output here and show `[...]` instead to make that clear? All this output clutters the article a bit in my opinion.
Maybe just show the JanusGraph ASCII art and the last line with `Channel started at port 8182` and remove everything else as these two lines show a) that JanusGraph Server is used and b) that it was started successfully?